README.md

```diff
@@ -5,7 +5,8 @@ This repository shows the example Botkube [source](https://docs.botkube.io/archi
 It is intended as a template repository to start developing Botkube plugins in Go. Repository contains:
 
 - The [`echo`](cmd/echo/main.go) executor that sends back the command that was specified,
-- The [`ticker`](cmd/ticker/main.go) source that emits an event each time the configured time duration elapses,
+- The [`ticker`](cmd/ticker/main.go) source that emits an event each time the configured time duration elapses
+- The [`forwarder`](cmd/forwarder/main.go) source that echos the message sent as an incoming webhook request,
```

Comment:
Some lines here have a comma [`,`] and some not. We should make them consistent.
I my opinion I wouldn't use commas to separate lines with bullet points.

Reply:
Agree - thank you, used periods for consistency as they are used everywhere in the document 👍